README.md

```diff
@@ -94,4 +94,25 @@ Test(<TestComponent/>)
 
 ~~~
 
+##mixin
+
+Use `.mixin` if you want to add new middleware as methods to `Test`. This gives a more natural way of using middleware:
+
+~~~js
+// In this example, CustomFind middleware was added to Test by mixin
+// and used if as it was a method on Test itself.
+
+Test(<TestComponent />)
```

Comment:
would be cool to do something like Test.mixin at the top of your tests file or in a setup file and then have them available everywhere.